RocketRL/python/func/ray_funcs.py

```diff
@@ -10,7 +10,97 @@
 import pandas as pd
 pd.set_option("display.max_columns",30)
 
+plotting = True
 
-os.chdir('/Users/ninalopatina/ray_results/default/DQN_CartPole-v0_0_2018-06-20_12-48-43rltozctp')
-df = pd.read_csv('progress.csv')
-df2 = pd.read_json('result.json',lines=True)
\ No newline at end of file
+
+WD = os.listdir()
+
+#!/usr/bin/env python
+from stat import S_ISREG, ST_CTIME, ST_MODE
+import os, sys, time
+
+import pickle
+import os
+import yaml
+
+import matplotlib.pyplot as plt
+
+# ##TO DO: get cfg in here more elegantly
+#set config path
+config_dir = 'config/'
+CWD_PATH = os.getcwd()
+config_path = os.path.join(CWD_PATH,config_dir,"config.yml")
+
+with open(config_path, 'r') as ymlfile:
+    cfg = yaml.load(ymlfile)
+
+# path to the directory (relative or absolute)
+dirpath = '/Users/ninalopatina/ray_results/RocketRL/'#sys.argv[1] if len(sys.argv) == 2 else r'.'
```

Comment:
Can we make this relative? This will break if ran in someone else's computer.

Reply:
This code is still in progress (not imported to main in this push), will incorporate this code & fix it before merging with the main branch